index.html

```diff
@@ -27,15 +34,8 @@
   except this little library grooves the <a
   href="https://github.com/facebook/react">React way</a>.</p>
 
-  <h2>Example</h2>
-
-  <iframe
-    width="100%"
-    height="300"
-    src="http://jsfiddle.net/L4z5wcx0/7/embedded/result,js,html,css/"
-    allowfullscreen="allowfullscreen"
-    frameborder="0">
-  </iframe>
+  <h2>Examples</h2>
+    <div id="react-waypoint-examples"></div>
```

Comment:
I'm kinda thinking that it would be better (slightly more linkable, clearer, likely easier to maintain) to have each example on its own, self-contained page that we link to here. What do you think?